components/Projects/ProjectCard/Avatar.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
+
+import Image from 'next/image';
+import { Tooltip } from 'react-tooltip';
+
+export interface AvatarData {
```

Comment:
I know that currently with don't have clear conventions but as a suggestion what do you think about having a suffix of `Props` to every props type?
for example in here, instead of `AvatarData` -> `AvatarProps`.